operations/research-and-development/engineering/team/README.md

```diff
@@ -38,13 +34,12 @@ Work for SET comes from various sources:
 3. The [~Sustained Engineering](https://community-daily.mattermost.com/core/channels/sustained-engineering) channel. Support may use this channel to ask questions or raise urgent issues.
 4. The [~Ask R&D](https://community-daily.mattermost.com/core/channels/ask-r-and-d) channel. This is the place where support and community can ask any type of R&D related questions.
 
-## Triage
-
-SET triages [SET tickets](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=33) asynchronously and also helps the Release Manager to
+When a new issue is raised, it is the SET primary's duty to:
 
-* Help route new unassigned tickets to the appropriate engineering team.
-* Assign bugs to SET when there is no clear engineering team owner.
-* Make sure reported bugs are in fact bugs and recommend turning non-bugs into stories or feature requests.
+1. Understand the customer impact and priority of the issue (if required with the help of the SET lead, or relevant product managers). If the issue is really a feature request or low-urgency bug, a JIRA ticket is filed and assigned to the appropriate team. In that case, the SET responsibilities end here. In case of higher-impact issues, the steps continue:
```

Comment:
Should we add specific criteria for what's an incident and what's a low-urgency bug? In the [incident response process doc](https://docs.google.com/document/d/1-AWQJQelgKvGVSP6sOIi9EOSVjxXVlJlwNuJlkcXKGA/edit#bookmark=id.4yjoevfvyytr) we have some criteria defined via severity but they're pretty cloud focused.

Reply:
I discussed this a bit with Sven, and I think adopting the same severity levels as support uses may make a lot of sense: https://mattermost.com/support/ I think we can also codify the steps (also around the hand-off) in a playbook, will work on that as well.